bin/tldr

```diff
@@ -4,7 +4,7 @@ const program = require('commander');
 const pkg = require('../package');
 const Tldr = require('../lib/tldr');
 const config = require('../lib/config');
-const platform = require('../lib/platform');
+const platformUtils = require('../lib/platformUtils');
```

Comment:
```suggestion
const Platform = require('../lib/platform');
```
Makes diff smaller and addresses name clash in a fairly standard manner.

Reply:
You think so? I mean to ask you in the extended commit message of 5e1b052af5b21f82b93e57bdc5caccf9203bddef

Reply:
Well, I've seen it more than once in various node codebases but that is anecdotal evidence. Maybe going with
```js
const platforms = require('../lib/platforms');
```
strikes the right balance?

Reply:
I don't have strong opinions regarding either, but do you think `platformUtils` it not standard?

Reply:
`utils` and `helpers` are well-known wildcards conveying absolutely zero meaning so some folks consider naming things using them as an antipattern. In this case, it has been used as name-clash avoiding sufix so it isn't really a problem excerpts it hurts my eyes 🙈
But let's take a step back, what exactly mandates rename in the first place, I failed to spot that name clash we are trying to avoid here? 😕

Reply:
I want to rename all internal references to `os` to `platform` where appropriate, but I couldn't easily in some places because of `const platform = require("..../platform.js")`, get it?

Reply:
That's the same situation as requiring the path module and then having a variable named path. In most cases platform is really accessed as a property of the program object which makes it a non-issue. And if there are some shadowing instances they should be addressed by renaming the local variable instead, for example, `targetPlatform` or `selectedPlatform`.

Reply:
Yeah, you're thinking the right way.  I guess I'm busy so I didn't think a lot about this.  I'll change the name back to `platform.js` 👍🏻

Reply:
What cases like this?
```javascript
function validatePlatform(platform) {
  if (platform && !platformUtils.isSupported(platform)) {
    return 'Unsupported platform : ' + platform;
  }
  return null;
}
```
Personally I'd pick `p` as a parameter name, but that doesn't strike my memory as conventional JS (I'm not working on much js lately)

Reply:
Well if you do validation then one thing is for sure, you are doing it on `input` 😉

Reply:
> Yeah, you're thinking the right way. I guess I'm busy so I didn't think a lot about this. I'll change the name back to `platform.js` 👍🏻

Watching me doing things you would soon realize that most of the time it seems like I'm not thinking at all 😂